dfc/target.go

```diff
@@ -486,6 +486,22 @@ func (t *targetrunner) httpobjput(w http.ResponseWriter, r *http.Request) {
 		}
 	} else {
 		// PUT
+		d := query.Get(URLParamDaemonID)
+		b := false
+		for id := range t.smap.Pmap {
+			if id == d {
+				b = true
+			}
+		}
+		for id := range t.smap.Tmap {
```

Comment:
When does target id appear in url? I see only proxy adds its id, so tmap lookup can be skipped.

Reply:
Yup, forgot to remove this part. It was there originally for `sendfile()` which goes from target-to-target. Thanks!